sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -244,164 +228,78 @@ public String getEntityPath() {
      * <p><strong>Complete a message with a transaction</strong></p>
      * {@codesnippet com.azure.messaging.servicebus.servicebusasyncreceiverclient.abandonMessageWithTransaction}
      *
-     * @param lockToken Lock token of the message.
+     * @param message The {@link ServiceBusReceivedMessage} message.
      * @param propertiesToModify Properties to modify on the message.
      * @param transactionContext in which this operation is taking part in. The transaction should be created first by
      * {@link ServiceBusReceiverAsyncClient#createTransaction()} or
      * {@link ServiceBusSenderAsyncClient#createTransaction()}.
      *
      * @return A {@link Mono} that completes when the Service Bus operation finishes.
-     * @throws NullPointerException if {@code lockToken}, {@code transactionContext} or
-     * {@code transactionContext.transactionId} is null.
+     * @throws NullPointerException if {@code transactionContext}, {@code message},
+     * {@link ServiceBusReceivedMessage#getLockToken()} or {@code transactionContext.transactionId} is null.
```

Comment:
It's not possible for lock token to be null because they cannot create a ServiceBusReceivedMessage instance on their own. Same with other instances of this.